pr_check.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="ocp-advisor"
+# IMAGE should match the quay repo set by app.yaml in app-interface
+export IMAGE="quay.io/cloudservices/ocp-advisor-frontend"
+export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build's workspace
+export APP_ROOT=$(pwd)
+# 16 is the default Node version. Change this to override it.
+export NODE_BUILD_VERSION=16
+COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+IQE_PLUGINS="ocp_advisor"
+IQE_MARKER_EXPRESSION="smoke"
```

Comment:
This will be `ui` or `ui and smoke` or something. We will need to decide. But it can be done later on.

Reply:
ok, I left just `ui`, is it ok for now?

Reply:
sure, nothing will be run, so it really doesn't matter (yet)